server/routes/index.ts

```diff
@@ -1,13 +1,42 @@
-import type { RequestHandler, Router } from 'express'
-
+import { RequestHandler, Router } from 'express'
 import asyncMiddleware from '../middleware/asyncMiddleware'
+import { Services } from '../services'
+import OtherRoutes from './otherRoutes'
+import auth from '../authentication/auth'
+import tokenVerifier from '../data/tokenVerification'
+import populateCurrentUser from '../middleware/populateCurrentUser'
+
+export default function Index({ userService, prisonerService, licenceService, communityService }: Services): Router {
+  const router = Router({ mergeParams: true })
 
-export default function routes(router: Router): Router {
   const get = (path: string, handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
+  // const post = (path: string, handler: RequestHandler) => router.post(path, asyncMiddleware(handler))
+
+  const otherAccessRoutes = new OtherRoutes(userService, prisonerService, licenceService, communityService)
 
-  get('/', (req, res, next) => {
-    res.render('pages/index')
+  const indexRoutes = () =>
+    get('/', (req, res) => {
+      res.render('pages/index')
+    })
+
+  const otherRoutes = () => {
+    get('/test/data', otherAccessRoutes.listTestData)
+    get('/staff/:username/detail', otherAccessRoutes.getStaffDetail)
+    get('/staff/:staffId/caseload', otherAccessRoutes.getStaffCaseload)
+    get('/prisoner/:nomsId/detail', otherAccessRoutes.getPrisonerDetail)
+  }
```

Comment:
This is the pin-phone method, which I liked, of defining routes as `functions of related routes`, and then defining these in their own modules. The `OtherRoutes` is a temporary split - so I'll probably split them based on the user roles - e.g. probation officer routes, prison approver routes, offender management unit routes etc..